app/jobs/concerns/hyrax/members_permission_job_behavior.rb

```diff
@@ -1,8 +1,6 @@
 module Hyrax
   # Grants read access for the supplied user for the members attached to a work
-  class MembersPermissionJob < ApplicationJob
-    queue_as Hyrax.config.ingest_queue_name
-
+  module MembersPermissionJobBehavior
```

Comment:
@bkeese I think we still need the queue_as here to have these jobs go into the ingest_queue. That or the statement needs to go back into each of the jobs.

Reply:
@elrayle Good point. I just assumed those statements were still in the jobs themselves and didn't look. Which place do you think makes more sense?

Reply:
I made a guess and decided to put it in the mixin.